tests/test_drafts.py

```diff
@@ -65,3 +65,16 @@ def test_delete_draft(api_client):
     draft.save()
     # ... and delete it for real.
     draft.delete()
+
+
+@responses.activate
+@pytest.mark.usefixtures("mock_draft_saved_response")
+def test_draft_version(api_client):
+    draft = api_client.drafts.create()
+    assert 'version' not in draft
```

Comment:
Hmm, so if a draft is created with the API it's first version should start at 0. Is this not the case in the SDK?